### ISSUE TITLE
publicIPSKU error with latest version

### ISSUE THREAD

**Issue by collinmcneese** [test-kitchen/kitchen-azurerm#193]
# Version:

[1.5.0]

# Environment:

```
driver:
  name: azurerm
  subscription_id: 'id'
  location: 'Central US'
  machine_size: 'Standard_D1'

transport:
  ssh_key: ~/.ssh/id_kitchen-azurerm

provisioner:
  name: chef_zero

verifier:
  name: inspec

platforms:
  - name: ubuntu-18.04
    driver:
      image_urn: Canonical:UbuntuServer:18.04-LTS:latest
      vm_name: kitchen-vm

suites:
  - name: default
    attributes:
```

# Scenario:

Trying to build VM with the latest driver and it provides an error re: publicIPSKU.  This is not specified in kitchen configuration but is included by default in the `internal.erb` template.

# Actual Result:

Error in Kitchen:
```
>>>>>>     Failed to complete #create action: [{
  "message": "MsRestAzure::AzureOperationError: InvalidTemplate: Deployment template validation failed: 'The template parameters 'publicIPSKU' in the parameters file are not valid; they are not present in the original template and can therefore not be provided at deployment time. The only supported parameters for this template are 'location, vmSize, newStorageAccountName, adminUsername, dnsNameForPublicIP, secretUrl, vaultName, vaultResourceGroup, existingStorageAccountBlobContainer, imagePublisher, imageOffer, imageSku, imageVersion, osDiskNameSuffix, vmName, nicName, storageAccountType, systemAssignedIdentity, userAssignedIdentities, bootDiagnosticsEnabled'. Please see https://aka.ms/arm-deploy/#parameter-file for usage details.'.",
```

**Comment by collinmcneese** [comment]
Merged in #194

**Comment by paulcalabro** [comment]
@collinmcneese  I came across that exact same error message above while using  1.5.2. Is this fixed?

**Comment by collinmcneese** [comment]
Yep, `1.5.2` still has the issue,  we need to do one more release on this and bump up to `1.5.3` to get the fixes from #197 in

**Comment by paulcalabro** [comment]
Thanks for confirming!
